bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -491,6 +491,13 @@ data:
     [[ -f "$REDIS_PASSWORD_FILE" ]] && export REDISCLI_AUTH="$(< "${REDIS_PASSWORD_FILE}")"
 
     if ! failover_finished; then
+        echo "I am the master pod and you are stopping me."
+        echo "Stopping client connections"
+        # Stopping client write connections to avoid data loss
+        run_redis_command CLIENT PAUSE "{{ mul (sub .Values.sentinel.terminationGracePeriodSeconds 10) 1000 }}" WRITE
+        # Running failover command
+        run_redis_command FAILOVER TIMEOUT "{{ mul (sub .Values.sentinel.terminationGracePeriodSeconds 10) 1000 }}"
```

Comment:
You do not need to run `CLIENT PAUSE WRITE` if you run `FAILOVER`. See docs: https://redis.io/commands/failover/

> The master will internally start a CLIENT PAUSE WRITE, which will pause incoming writes and prevent the accumulation of new data in the replication stream.

Reply:
Good catch! I just updated the code accordingly.